package.json

```diff
@@ -16,19 +16,19 @@
   },
   "schematics": "./src/collection.json",
   "dependencies": {
-    "@angular-devkit/core": "^15.0.0-next.5",
-    "@angular-devkit/schematics": "^15.0.0-next.5",
-    "@schematics/angular": "^15.0.0-next.3",
-    "typescript": "~4.8.1"
+    "@angular-devkit/core": "^19.2.14",
+    "@angular-devkit/schematics": "^19.2.14",
+    "@schematics/angular": "^19.2.14",
+    "typescript": "^5.8.3"
```

Comment:
Only because the `typescript` package doesn't follow semver, I typically lock this to minor versions with `~` to be safe